pygotham/about/models.py

```diff
@@ -0,0 +1,29 @@
+"""About models."""
+
+from slugify import slugify
+from sqlalchemy_utils.decorators import generates
+
+from pygotham.core import db
+
+__all__ = ('AboutPage',)
+
+
+class AboutPage(db.Model):
```

Comment:
Agreed. I think there are also about pages that are not event-specific, e.g. "About the Conference" and "Code of Conduct". In lieu of having to copy these over year after year, should we allow `null` for `event_id` and consider "current" about pages to be ones with the correct event or none?